modules/serverless-nodejs-template.adoc

```diff
@@ -6,7 +6,7 @@
 [id="serverless-nodejs-template_{context}"]
 = Node.js function template structure
 
-When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project, with the exception of an additional `func.yaml` configuration file.
+When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project. The only exception is the additional `func.yaml` configuration file, used for specifying the image.
```

Comment:
```suggestion
When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project. The only exception is the additional `func.yaml` configuration file, which is used for specifying the image.
```

Reply:
@jeana-redhat Aren't we supposed to have two or more sentences in the first paragraph?

Reply:
Nvm, meant to comment another suggestion